.github/workflows/nightly_snapshot.yaml

```diff
@@ -310,8 +310,7 @@ jobs:
             cp -a ../target/doc/* snapshots/$target_branch/docs/rust/
 
             # Fix up link to Slint language documentation
-            sed -i "s!https://slint-ui.com/releases/.*/docs/slint!../../slint!" snapshots/$target_branch/docs/rust/slint/*.html
-            sed -i "s!https://slint-ui.com/releases/.*/docs/tutorial!../../tutorial!" snapshots/$target_branch/docs/rust/slint/*.html
+            sed -i "s!https://slint-ui.com/releases/.*/docs/!../../!" snapshots/$target_branch/docs/rust/slint/*.html
```

Comment:
btw, this is not only run for the snapshots, but also the actual release  (because  snapshots/master is copied into releases/x.y.z as part of the release script)

But relative link is alright anyway on both the release and the snapshot